crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1471,9 +1469,12 @@ fn visit_node(
         | figma_schema::NodeData::Line { vector }
         | figma_schema::NodeData::RegularPolygon { vector }
         | figma_schema::NodeData::Star { vector }
-        | figma_schema::NodeData::Vector { vector } => {
-            ViewShape::Path { path: fill_paths, stroke: stroke_paths, is_mask: vector.is_mask }
-        }
+        | figma_schema::NodeData::Vector { vector } => ViewShape::Path {
+            path: fill_paths,
+            stroke: stroke_paths,
+            stroke_cap: node.stroke_cap.clone().into(),
```

Comment:
Weirdly I think stroke cap is a style thing, but up to you if you want to keep it here.

Reply:
I'll leave it for now, since it's also in the `Arc` shape.